### PR TITLE
redo refactor guides

### PR DESCRIPTION
**Pull request by geoffcline**
**1. Issue, if available:**
redo of #1428 

**2. Description of changes:**
refactor getting started guides to have a parent guide directory

**3. How was this change tested?**
netlify preview

**4. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by geoffcline** [comment]
don't merge yet, doing a broken URL check on the netlify preview

**Comment by geoffcline** [comment]
I used `git mv` instead of `mv` in hopes of making the diff clearer, no luck.

**Comment by geoffcline** [comment]
ready 2 review

File: website/content/en/preview/getting-started/getting-started-with-terraform/_index.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ terraform apply -var "cluster_name=${CLUSTER_NAME}"
 
 The global log level can be modified with the `logLevel` chart value (e.g. `--set logLevel=debug`) or the individual components can have their log level set with `controller.logLevel` or `webhook.logLevel` chart values.
 
-### Provisioner
+### provi
```
Comment (njtran) [inline website/content/en/preview/getting-started/getting-started-with-terraform/_index.md:289]: Can you fix this?

Comment (geoffcline) [reply website/content/en/preview/getting-started/getting-started-with-terraform/_index.md:289]: done

**Review by spring1843 — APPROVED** [review]
LGTM

**Comment by ellistarn** [comment]
Looks like we have a rebase conflict again.

**Pull request closed without merging**